### DISCUSSION TITLE
Podman Home Assistant Container LXC Samba share issue.

### DISCUSSION THREAD

**Discussion by love768952693** [tteck/Proxmox#523] [Ideas]
First of all, I would like to thank the developers for providing convenient and fast usage methods.

I want to know how to quickly share and access the configuration directory through a container such as the Podman Home Assistant Container LXC, similar to the Samba share of the supervisor in the OS system.

In the OS version, Samba shares can be loaded through the supervisor to facilitate the management and modification of files. The disadvantage is that it takes a lot of resources, so I want to use containers.

**Comment by tteck** [comment]
I'd suggest installing File Browser.
